doc/source/examples/incompressible-flow/3d-taylor-green-vortex/3d-taylor-green-vortex.rst

```diff
@@ -0,0 +1,373 @@
+
+The ``smoother``, ``Eigenvalue estimation parameters`` and ``coarse-grid solver`` subsections are explained in the **Theory Guide** (under construction).
+
+
+
+----------------------
+Running the Simulation
+----------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``lethe-fluid`` or ``lethe-fluid-matrix-free`` executables are within your path, the matrix-based simulation scan be launched by typing:
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np n_proc lethe-fluid tgv-matrix-based.prm
+
+and the matrix-free simulations can be launched by typing
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np n_proc lethe-fluid-matrix-free tgv-matrix-free.prm 
+
+For a 5 initial refinement (:math:`32^3` Q2 cells), the matrix-based solver takes around 1 hour and 20 minutes on 16 cores while the matrix-free solver takes less than 20 minutes. Running the same problem, but in Q3 (:math:`32^3` Q3 cells), the matrix-free solver takes less than 2 hours while the matrix-based solver takes close to a day and consumes a tremendous amount of ram (approx. 80 GB). If you have 64 GB of ram, you can run an even finer mesh (:math:`64^3` Q3 cells) using the matrix-free solver in approximately 16 hours.
+
+
+----------------------
+Results and Discussion
+----------------------
+
+The flow patterns generated by the Taylor-Green vortex are quite complex. The following animation displays the evolution of velocity iso-contours as the vortex break downs and generates smaller structures.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------+
+| .. raw:: html                                                                                                                                      |
+|                                                                                                                                                    |
+|    <iframe width="520" height="400" src="https://www.youtube.com/embed/GGij2g_yz5g?si=Q81lcHyPd9ONxqT4"  frameborder="0" allowfullscreen></iframe> |
+|                                                                                                                                                    |
++----------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+Using the ``enstrophy.dat`` and ``kinetic_energy.dat`` files generated by Lethe, the temporal decay of the kinetic energy can be monitored. First, we calculate the time-derivative of the kinetic energy by invoking the first script present in the example folder:
+
+.. code-block:: text
+  :class: copy-button
+
+  python3 calculate_dissipation_rate.py -i kinetic_energy.dat -o output.dat
```

Comment:
I would rather not do that because that depends on the path you  are in... It was just to show an example of how to launch it, I'm not expecting people to copy paste the full command...